spec/fixtures/bg_slide/ppt/presentation.xml

```diff
@@ -6,6 +6,6 @@
   <p:sldIdLst>
     <p:sldId id="256" r:id="rId2"/>
   </p:sldIdLst>
-  <p:sldSz cx="12192000" cy="6858000"/>
+  <p:sldSz cx="13004800" cy="9753600"/>

```

Comment:
I understand that LSB will generate slides that are 1024x768, but what value does it add to change all these fixtures in the PPTX gem?

It's probably _slightly_ better for the gem bake in the same assumption for default slide side that PowerPoint does.

I think LSB should specify a slide size of 1024x768 when it constructs presentations. Is that hard to do now? If so, what would have to change about the PPTX gem so that it'd be easy to pass the desired dimensions in to a new Presentation object?